dgen_os/python/input_data_functions.py

```diff
@@ -205,24 +232,33 @@ def import_table(scenario_settings, con, engine, role, input_name, csv_import_fu
     
     Parameters
     ----------
-    scenario_settings : 'SQL schema'
-        Schema in which to look for the scenario settings
-    con : 'SQL connection'
+    scenario_settings : :class: `settings.ScenarioSettings`
+        Custom object in which to look for the scenario settings
+
+    con : :class: `psycopg2.connection`
         SQL connection to connect to database
-    engine : 'SQL engine'
-        SQL engine to intepret SQL query
-    role : 'string'
+
+    engine : :class: `sqlalchemy.Engine`
+        SQL engine to intepret SQL query 
+
+    role : `string`
         Owner of schema
-    input_name : 'string'
-        Name of the csv file that should be imported     
-    csv_import_function : 'function'
+
+    input_name : `string`
+        Name of the csv file that should be imported   
+
+    csv_import_function : `function`
         Specific function to import and munge csv 
```

Comment:
```suggestion
        Specific dGen convenience function in `input_data_functions` to use for importing and processing of scenario CSV datasets
```
Updating this to better describe what the "function" is